README.md

```diff
@@ -51,6 +58,10 @@ poetry run isort -y
 poetry run mypy --show-error-codes mitol/
 ```
 
+To run the full test suite against all python/django versions, you should run `tox` instead of `pytest` directly.
```

Comment:
Should we update the `poetry run pytest` line above to use tox instead?

Reply:
I think running poetry by itself is still very valid, it runs in a few seconds. The entire test matrix for some apps (particularly the mail app) can take much longer to run because of the test matrix size.